styles.css

```diff
@@ -153,4 +626,161 @@
   .social li a i {
     align-items: center;
   }
+
+  .works .container {
+    padding: 0 10px;
+  }
+
+  .about-me .container2 img {
+    width: 40%;
+    position: absolute;
+    bottom: 0;
+    right: 0;
+  }
+
+  .works .container .project img {
+    display: flex;
+    justify-content: flex-start;
+    width: 150%;
+    margin-bottom: 24px;
+  }
+
+  .works .container .project h2 {
+    width: 327px;
+    height: 36px;
+    font-family: 'Crete Round', sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 32px;
+    line-height: 44px;
+    color: #091e42;
+    margin-bottom: 12px;
+  }
+
+  .works .container .project2 h2 {
+    font-size: 32px;
+    display: flex;
+    align-items: center;
+    color: #fff;
+    margin-bottom: 12px;
+    margin-left: 10px;
+  }
+
+  .about-me .container2 p {
+    font-size: 22px;
+    line-height: 34px;
+    color: #f4f5f7;
+    margin-top: 24px;
+    margin-bottom: 20px;
+    width: 500px;
+  }
+
+  .works .container .project2 p {
+    font-size: 16px;
+    line-height: 24px;
+    color: #fff;
+    margin-left: 10px;
+    margin-top: 16px;
+    margin-bottom: 38px;
+  }
+
+  .works .project2 .btn2 {
+    display: flex;
```

Comment:
- [ ] Kindly use `display: grid` instead of  `display: flex` for `works` class only because it is a requirement :slightly_smiling_face: 

Check below and also click this [link](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/2_mobile.md) for more information.
>  In order to lay out the elements in the "Works" section you should use CSS Grid and Flexbox combined (it is mandatory that you use both). Use CSS Grid in the main container that contains all of the cards and use Flexbox to lay out an individual card.